src/stream/src/executor/locality_provider.rs

```diff
@@ -740,7 +743,8 @@ impl<S: StateStore> LocalityProviderExecutor<S> {
                     .inc_by(cur_barrier_upstream_processed_rows);
 
                 yield Message::Barrier(barrier);
-                post_commit.post_yield_barrier(None).await?;
+                post_commit1.post_yield_barrier(None).await?;
```

Comment:
`commit_assert_no_update_vnode_bitmap` doesn't mean no-op, but means that there must no vnode update bitmap happen. 

Since here we always pass `None` to `post_yield_barrier`, we can assert that there is no update vnode bitmap, we therefore we can call `commit_assert_no_update_vnode_bitmap` directly.

Reply:
cc @shanicky Do you have any idea about how to deal with the online scaling in the locality backfill operator?